Firmware/Marlin_main.cpp

```diff
@@ -2058,6 +2106,8 @@ void process_commands()
   float tmp_motor_loud[3] = DEFAULT_PWM_MOTOR_CURRENT_LOUD;
   int8_t SilentMode;
 #endif
+  KEEPALIVE_STATE(IN_HANDLER);
```

Comment:
@Sebastianv650 If you have time to test, can you try and comment out this line, and see if that fixes the constant busy signal?